### ISSUE TITLE
Navigation filters broken

### ISSUE THREAD

**Issue by robdekort** [statamic/cms#8087]
### Bug description

Filters in the Navigation Add Entry Stack. Clicking on them doesn't do anything (tested in Chrome and Safari).
<img width="1309" alt="Screenshot 2023-05-10 at 09 55 44" src="https://github.com/statamic/cms/assets/69107412/ca155fad-cdc0-4d3a-8a4a-0bfc17086c34">

### How to reproduce

Create a navigation
Add an entry to the navigation
Try the filters

### Environment

```yaml
Environment
Laravel Version: 9.52.7
PHP Version: 8.2.3
Composer Version: 2.2.7
Environment: local
Debug Mode: ENABLED
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: statamic
Database: mysql
Logs: stack / single
Mail: smtp
Queue: sync
Session: file

Statamic
Addons: 5
Antlers: runtime
Stache Watcher: Enabled
Static Caching: Disabled
Version: 4.0.0 PRO

Statamic Addons
rias/statamic-color-swatches: 2.2.0
studio1902/statamic-peak-browser-appearance: 2.0
studio1902/statamic-peak-commands: 2.0
studio1902/statamic-peak-seo: 2.0
studio1902/statamic-peak-tools: 3.0
```

### Installation

Fresh statamic/statamic site via CLI

### Antlers Parser

None

**Comment by SirNovi** [comment]
Just ran into the same issue and verified it on a fresh, clean 4.0.0 install. 
Buttons are not working, no errors in console.

**Comment by duncanmcclean** [comment]
Are you able to test this again on the latest version of Statamic? I can click on the filter buttons fine & the filtering works:

<img width="1285" alt="image" src="https://github.com/statamic/cms/assets/19637309/870e446b-7f83-43f9-9442-08bccb25a070">

**Comment by robdekort** [comment]
Nice. This indeed seems fixed. Maybe by some of the recent popover changes? However, not the status pills. What's interesting is that you have them in your screenshot but for me it's just a bland (and the first) text column.

**Comment by jesseleite** [comment]
@robdekort Just the status pill styling in the table itself you mean? Everything is functional on latest version? Screenshot?

**Comment by duncanmcclean** [comment]
> Nice. This indeed seems fixed. Maybe by some of the recent popover changes? However, not the status pills. What's interesting is that you have them in your screenshot but for me it's just a bland (and the first) text column.

I have a PR open to fix the status pills (which you've logged as another issue): #8210

**Comment by jasonvarga** [comment]
Looks like the filters were fixed in #8103.

**Comment by robdekort** [comment]
> I have a PR open to fix the status pills (which you've logged as another issue): #8210

Oh nice, can't remember all the issues I reported :-). Great this is all solved now.
